cssom-view-1/Overview.bs

```diff
@@ -1136,20 +1132,17 @@ The <dfn attribute for=CaretPosition>offset</dfn> attribute must return the <a>c
 The <dfn method for=CaretPosition>getClientRect()</dfn> method must follow these steps,
 aborting on the first step that returns a value:
 
-1. If <a>caret range</a> is not null:
-    1. Let <var>list</var> be the result of invoking the {{Range/getClientRects()}} method on the range.
-    1. If <var>list</var> is empty, return null.
-    1. Return the {{DOMRect}} object in <var>list</var> at index 0.
 1. If <a>caret node</a> is a text entry widget that is a replaced element,
     and that is in the document,
     return a [=scaled=] {{DOMRect}} object for the caret in the widget
     as represented by the <a>caret offset</a> value.
     The <a>transforms</a> that apply to the element and its ancestors are applied.
-1. Return null.
+1. Otherwise:
 
-Note: This {{DOMRect}} object is not <a spec=html>live</a>.
+    1. Let <var>caretRange</var> be a collapsed {{Range}} object whose [=range/start node=] and [=range/end node=] are set to <a>caret node</a>, and whose [=range/start offset=] and [=range/end offset=] are set to <a>caret offset</a>.
+    1. Return the {{DOMRect}} object which is the result of invoking the {{Range/getBoundingClientRect()}} method on <var>caretRange</var>.
 
-Issue(10230): Consider removing <a>caret range</a> concept from <a>caret position</a> interface.
+Note: This {{DOMRect}} object is not <a spec=html>live</a>.
```

Comment:
Should we expand on this and clarify that the `CaretPosition` itself, not just the `DOMRect` is not live?

Reply:
Sure.

Reply:
Done.